config/webpack/environment.js

```diff
@@ -1,30 +1,30 @@
-const { environment } = require('@rails/webpacker');
-const erb = require('./loaders/erb');
-const webpack = require('webpack');
+const { environment } = require('@rails/webpacker')
+const erb = require('./loaders/erb')
+const webpack = require('webpack')
 
 environment.config.merge({
   resolve: {
     alias: {
-      Circles: 'themes/current/vendor/circles/circles.min',
-    },
-  },
-});
+      Circles: 'themes/current/vendor/circles/circles.min'
+    }
+  }
+})
 
 environment.plugins.prepend(
   'Provide',
   new webpack.ProvidePlugin({
     jQuery: 'jquery',
-    '$': 'jquery',
+    $: 'jquery',
```

Comment:
Why was this changed?

Reply:
Dunno. My guess is the JavaScript formatter did this?

Reply:
Verified. Prettier is updating the format here.